src/codeu/chat/client/commandline/Panel.java

```diff
@@ -48,10 +49,13 @@ public void register(String commandName, Command command) {
   // will be returned. True will be return if a command is found. Whether or not
   // the command was successful is not returned.
   //
-  public boolean handleCommand(String commandName, Scanner line) {
```

Comment:
Since this change modifies how the command from the command line is parsed and passed to the command interface, maybe a brief comment should be inserted to clarify the change made to this function.